src/main/webapp/app/core/core.module.ts

```diff
@@ -64,7 +63,6 @@ import { fontAwesomeIcons } from './icons/font-awesome-icons';
 export class JHipsterRegistryCoreModule {
   constructor(iconLibrary: FaIconLibrary, dpConfig: NgbDatepickerConfig) {
     registerLocaleData(locale);
-    iconLibrary.addIconPacks(fas);
```

Comment:
@DanielFran I have tested the registry with microservices connected, some icons doesn't show in the navbar

Reply:
This is related with https://github.com/jhipster/generator-jhipster/pull/11079 from @ruddell .
Will rollback that changes...

Reply:
I fixed the missing fontawesome icons, they must be imported in the font-awesome-icons.ts file